base_station/gui/src/components/AutonWaypointEditor.vue

```diff
@@ -133,6 +133,9 @@ export default {
 
   created: function () {
 
+    this.toggleTeleopMode(false);
+    this.toggleAutonMode(false);
```

Comment:
Added to keep Auton from starting with Teleop enabled (which is critical since the checkbox for teleop will start disabled)